Assets/LeapMotion/Modules/HierarchyRecording/Scripts/LeapRecording/VectorHandRecording.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Leap.Unity.Query;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Recording {
+
+  public class VectorHandRecording : LeapRecording {
+
+    [SerializeField]
+    private VectorFrameCurves _frameCurves;
+
+    private Hand _leftHand = new Hand();
+    private Hand _rightHand = new Hand();
+
+    public override float length {
+      get {
+        return _maxKeyframeTime - _minKeyframeTime;
+      }
+    }
+
+    private float _minKeyframeTime = 0f;
```

Comment:
Do these min and max frame times need to be serialized?  As far as I can tell they will always end up being zero.